### ISSUE TITLE
1 channel vs 3 channel performance

### ISSUE THREAD

**Issue by suman-19** [AlexeyAB/darknet#3960]
Will training on 1 channel images improve performance in terms of inference time and fps ?

**Comment by AlexeyAB** [comment]
The less model, the more improvement will be achieved.

**Comment by suman-19** [comment]
I don't think that the code is optimized for single channel images. Can you please confirm the changes required for this kind of optimization ..?
